Lib/test/test_sys_settrace.py

```diff
@@ -2064,6 +2064,40 @@ def test_jump_simple_backwards(output):
         output.append(1)
         output.append(2)
 
+    @jump_test(1, 4, [5])
+    def test_jump_is_none_forwards(output):
+        x = None
+        if x is None:
+            output.append(3)
+        else:
+            output.append(5)
+
+    @jump_test(6, 5 [3, 5, 6])
```

Comment:
Heads-up: missing comma!

```suggestion
    @jump_test(6, 5, [3, 5, 6])
```